doc/method/results.md

```diff
@@ -0,0 +1,342 @@
+
+# Results
+
+## Alignment
+
+The alignment procedure examines every source node.  The following is
+a breakdown of the disposition of source nodes, across all sources, as
+OTT is assembled.  Explanations of the categories follow.
+
+```
+     49  curated alignment
+    105  align to skeleton taxonomy
+
+  21584  disjoint divisions
+    172  disparate ranks
+  25800  by lineage
+   7653  overlapping membership
+    219  same division
+  83012  by name
+
+2325870  confirmed
+   1767  by elimination
+   8592  ambiguous tip
+    452  ambiguous internal
+  10678  disjoint membership
+    921  disjoint membership
+3757548  not aligned
+
+6244422  total source taxon records
+```
+
+
+_curated alignment:_ Some alignments are hand-crafted, usually to
+repair mistakes made by automatic alignment.  
+
+_align to skeleton taxonomy:_ Alignments to the skeleton (for
+'division' calculations) are performed before the main alignment loop
+begins.
+
+_disjoint divisions_, ..., _by name_:
+Automated source record alignments, broken down according to which
+heuristic (see methods) was responsible for narrowing the candidate
+set down to a single union record.
+
+_confirmed_: There was only a single candidate, and it was confirmed
+by a 'yes' answer from one of the heuristics (usually same name).
+
+_by elimination_: Only a single candidate, but not confirmed by any
+heuristic (match involved a synonym).
+
+_ambiguous_: The heuristics were unable to choose from among multiple
+candidates; no alignment is recorded for the source node.
+
+_ambiguous tip_: Ambiguous, and the source node is a tip.
+
+_ambiguous internal_: Ambiguous, and the source node is an internal
+node (has children).
+
+_not aligned_: There were union taxonomy no candidates at all for this
+source taxon.  The source taxon name is new at this point in assembly.
+
+
+   * KC: do certain heuristics work better / worse for different types of problems?
+     [how would one assess this ?? what are examples of 'types of problems'?]
+
+
+## Merge
+
+The merge phase examines every source node, copying unaligned source
+nodes into the union taxonomy when possible.  The following table
+categorizes the fate of each source node during the merge phase.
+
+```
+2162104  aligned tip
+ 304121  aligned internal node
+3482704  new tip
```

Comment:
will change "not aligned" to "unaligned."

to make the two cases you quote parallel one would need something along the lines of:
  - no new tip (because aligned)
  - new tip (because unaligned and not ambiguous)
etc.